userguide/storage.rst

```diff
@@ -138,9 +136,12 @@ left arrow to return it to the :guilabel:`Available Disks` section.
 To add multiple :guilabel:`Data VDevs`, click :guilabel:`Add Data` for
 each required additional vdev.
 
-Alternately, click the :guilabel:`SUGGEST LAYOUT` button which will
-add all of the disks and suggest an optimal layout for both redundancy
-and capacity.
+Click :guilabel:`RESET LAYOUT` to return all disks to the
```

Comment:
This might be better as a "can" scenario. A user may not always want to reset his layout of disks. Suggestion:

To return all disks to the :guilabel:`Available Disks` area and close all but one :guilabel:`Data VDevs` table, click :guilabel:`RESET LAYOUT`.